libs/output-mapping/src/Storage/NativeTypeDecisionHelper.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Storage;
+
+use Keboola\StorageApiBranch\StorageApiToken;
+
+class NativeTypeDecisionHelper
+{
+    public static function shouldEnforceBaseTypes(StorageApiToken $token, string $backend): bool
+    {
+        return !$token->hasFeature('bigquery-native-types') && $backend === 'bigquery';
```

Comment:
Jestli jsi s timhle nazvem featury ok, vyslu PR na KBC. Ted zaregistrovana zatim neni.

Reply:
jo za mě může být

Reply:
https://github.com/keboola/connection/pull/5319

Reply:
Pridani feature je uz mergnuty ale nasadi se az rano. Tady to nic neblokuje. Projektum to ted stejne zapinat nebudeme a testy si mockuji token.

Reply:
Feature je pridana do KBC a popsal jsem i ve feature listu.